keysign/app.py

```diff
@@ -171,61 +174,54 @@ def on_activate(self, app):
 
         window.show_all()
         self.add_window(window)
-
+        reactor.run()
```

Comment:
I'm a bit confused to see the run method here.
I somehow expect the `run` call to be close to the GLib `run` call. In this case I would expect `reactor.run` to be adjacent to the `run` call of the super class.

In general I'm a bit confused with the two main loop concepts.  Do we need to call both, `reactor.run` and `GtkApplication.run`?  It doesn't seem to hurt doing it, but it's creating knots in my brain.

The same goes for quitting the main loop.